airflow/www/utils.py

```diff
@@ -471,9 +471,12 @@ def clean_column_names():
 
     def is_utcdatetime(self, col_name):
         from airflow.utils.sqlalchemy import UtcDateTime
-        obj = self.list_columns[col_name].type
-        return isinstance(obj, UtcDateTime) or \
-            isinstance(obj, sqla.types.TypeDecorator) and \
-            isinstance(obj.impl, UtcDateTime)
+
+        if col_name in self.list_columns:
```

Comment:
What was this change for btw?

Reply:
This function is trying to get "type" from the new field tag, while it's not really a column but a relationship.